### PR TITLE
Update animator root scroller definition

### PR DESCRIPTION
**Pull request by majido**
Update animator root scroller definition to allow the root itself to be it.

Also
 - add a note to fix the currentTime to match ScrollTimeline
 - update note to make clear we mean pointer input as scroll input is supported

### CODE REVIEW

**Comment by majido** [comment]
@flackr PTAL

File: index.bs [hunk 1]
```diff
@@ -431,12 +431,17 @@ The timelines present in an animator's <a>animator timeline list</a> can also af
...
         of each animation frame.
     * If the string "scroll" is present in an animator's <a>animator timeline list</a>, then when
-        the scroll position of the nearest non-visible overflow ancestor to the animator’s
-        <a>root element</a> changes, that animator's <a>animation requested flag</a> must be set to
-        <a>frame-requested</a>.
+        the scroll position of the animator <a>root scroller</a> changes, that animator's
+        <a>animation requested flag</a> must be set to <a>frame-requested</a>.
+
+Issue: According to the current ScrollTimeline spec, the currentTime should get normalized against
+ the animation duration. We shoud match the same semantic here.
```
Comment (flackr) [inline index.bs]: This was actually fixed in Stephen's last commit which unfortunately I didn't catch went into the old spec-wip branch. I've merged it into the main gh-pages branch so you should be able to see it now. We simply construct a ScrollTimeline and defer to it to generate the currentTime.

**Review by flackr — APPROVED** [review]

**Comment by yoavweiss** [comment]
@majido - Could you join the WICG to appease the IPR bots?

**Comment by majido** [comment]
@yoavweiss  I was actually a member of WICG in my w3c account but had to link my github account with my w3c account. It should good now. :+1:

**Pull request closed without merging**